docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -142,7 +133,7 @@ Tax estimates will be requested, depending on the BigCommerce store's settings,
 
 Estimate requests are not expected after the following events.
 
-* While browsing a store's product catalog or product pages.
+* While browsing a store's home page, product catalog or product pages.
```

Comment:
Clarifying that the home page is also not ever an expected place that a estimate request will occur (on the back of a ticket that identified an edge case where it was).

Reply:
👍